src/sdk/namespace_fs.js

```diff
@@ -968,7 +971,7 @@ class NamespaceFS {
             const stat = await file.stat(fs_context);
             this._throw_if_delete_marker(stat);
             // await this._fail_if_archived_or_sparse_file(fs_context, file_path, stat);
-
+            this._handle_read_part_number(params, stat);
```

Comment:
when file is a directory object the stat here is taken from the .folder file and not from the md path, therefore we won't have the xattr, we need to add another stat for having this info.